vignettes/nldi.Rmd

```diff
@@ -0,0 +1,336 @@
+---
+title: "Network Linked Data Index: `dataRetrieval`"
+author: "[Mike Johnson](mikecp11@gmail.com)"
+date: '`r Sys.Date()`'
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    toc_level: 3
+    fig_caption: yes
+    fig_height: 7
+    fig_width: 7
+vignette: >
+  %\VignetteIndexEntry{Network Linked Data Index: `dataRetrieval`}
+  \usepackage[utf8]{inputenc}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+
+```{r setup, include=FALSE}
+library(knitr)
+library(dataRetrieval)
+
+opts_chunk$set(warning = FALSE, message = FALSE, out.width = "75%", fig.align = 'center', comment  = "#>")
+```
+
+This post will demonstrate `dataRetrieval` functions to query the [Network Linked Data Index](https://labs.waterdata.usgs.gov/about-nldi/index.html) (NLDI).
+
+The NLDI provides a information backbone to navigate the NHDPlusV2 network and discover features indexed to the network. This process of feature discovery is very natural. In fact, many of us use it every day. 
+
+***** 
+
+## Navigation Analogy
+
+Imagine you want to know where UCSB is. For this we might use Google Maps. In asking about the entity 'UCSB' we get information related to the feature (links, operating hours, photos) and things nearby.
+
+<br><br>
+
+```{r, fig.cap = "UCSB via Google Maps",  echo = FALSE}
+knitr::include_graphics("ucsb.png")
+```
+<br><br>
+
+Imagine then that you want to travel from UCSB to UCLA and know you need gas along the way. In Google Maps you might use UCSB at your **origin** and ask for directions to UCLA that include gas stations along your route.
+
+<br><br>
+```{r, fig.cap = "Gas stations between UCSB and UCLA",  echo = FALSE}
+include_graphics("nldi-gas-station-example.png")
+```
+<br><br>
+
+
+
+The NLDI works in a very similar way allowing users to discover known features, navigate from those along a route, and find other features on that route. The primary difference being navigation occurs over the hydrologic network rather then a transportation network. As such routes are not free-form an instead based on hydrologic topology - that is you can travel "up" or "down" stream.
+
+The NLDI is becoming a more robust (in terms of performance) and ubiquitous tool. Notably it is driving the mapping interface of the of NWIS "Next Generation Station Pages" (see below), and provides a way to interface with "big" data through web-based retrievals. This service is now available as part of the `dataRetrieval` package through the `findNLDI` function.
```

Comment:
Could probably drop this and put a note in "NEWS.md" to this affect.

Reply:
put a blurb in news, but for now like this I think (otherwise would remove the intro analogy)